sips/adapt_fees_to_declared_effective_balance.md

```diff
@@ -0,0 +1,166 @@
+| Author | Title          | Category                             | Status  | Date               |
+|-------|----------------|--------------------------------------|---------|--------------------|
+| Gal Rogozinski & Marco Tabasco | Adapt fees to declared effective balance | Contracts & Core | open-for-discussions  |18.09.2024 |
+
+
```

Comment:
what bothers me with this approach is that a user having thousands of validators will need to update each one independently .. 

Post pectra all validators start to compound above 32 automatically? if so it will be a nightmare from UX POV

Reply:
Could be also that updating clusters instead of validators will simplify things